android_tests/lib/android/specs/install.rb

```diff
@@ -5,11 +5,11 @@ def pkg
   end
 
   def installed
-    is_installed?(pkg).must_equal true
+    installed?(pkg).must_equal true
```

Comment:
Does this work? I think [`is_installed?`](https://github.com/appium/ruby_lib/blob/ad10640c58ba1435b32a4d3a8268f66ae4c4b74e/lib/appium_lib/device/device.rb#L144) is the proper method name. I don't see a definition for `installed?`